packages/lodestar/src/api/impl/validator/validator.ts

```diff
@@ -139,14 +143,26 @@ export class ValidatorApi implements IValidatorApi {
       return  state.validators.findIndex((v) => this.config.types.BLSPubkey.equals(v.pubkey, publicKey));
     }));
 
-    return validatorIndexes.map((validatorIndex) => {
+    const attesterDuties = validatorIndexes.map((validatorIndex) => {
       return assembleAttesterDuty(
         this.config,
         {publicKey: state.validators[validatorIndex].pubkey, index: validatorIndex},
         state,
         epoch
       );
     });
+    const subnets: Set<string> = new Set();
+    attesterDuties.forEach(duty => subnets.add(getCommitteeIndexSubnet(duty.committeeIndex)));
+    for (const subnet of subnets) {
+      const peerIds = this.reps.getPeerIdsBySubnet(subnet);
+      if (peerIds.length < 3) {
```

Comment:
The spec does not specify a constant, I think we should have a configuration variable here. Any ideas?